tests/unit/s2n_session_ticket_test.c

```diff
@@ -1111,8 +1121,109 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_config_free(client_config));
     }
 
+    EXPECT_SUCCESS(s2n_reset_tls13());
+
+    /* Session resumption APIs and session_ticket_cb return sane values
+     * when receiving a new ticket in TLS1.3
+     * TODO: https://github.com/aws/s2n-tls/issues/2553
```

Comment:
hmm this is more like set-up for a test, rather than a test itself. Also, why are we adding the todo here? Didn't this test complete the todo?

Reply:
The todo is actually for a different task: https://github.com/aws/s2n-tls/issues/2553
We have a TLS1.2 version of that test (https://github.com/aws/s2n-tls/blob/main/tests/unit/s2n_session_ticket_test.c#L1063-L1066), but can't make any meaningful TLS1.3 assertions until that issue is addressed.

Reply:
Alright that works for me